components/style/values/specified/image.rs

```diff
@@ -372,21 +394,22 @@ impl ToCss for AngleOrCorner {
     }
 }
 
-impl Parse for AngleOrCorner {
-    fn parse(context: &ParserContext, input: &mut Parser) -> Result<Self, ()> {
-        if input.try(|input| input.expect_ident_matching("to")).is_ok() {
+impl AngleOrCorner {
+    fn parse(context: &ParserContext, input: &mut Parser, mode: CompatMode) -> Result<Self, ()> {
+        if let Ok(angle) = input.try(|i| Angle::parse(context, i)) {
```

Comment:
AFAIK the same is true for non-webkit gradients and thus is a bit unrelated to that change.